lib/v2/huxiu/briefColumn.js

```diff
@@ -0,0 +1,43 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate, parseRelativeDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const { id } = ctx.params;
+    const link = `https://api-brief.huxiu.com/briefColumn/getContentListByCategoryId`;
+    const response = await got
+        .post(link, {
+            headers: {
+                'Content-Type': 'application/x-www-form-urlencoded',
+            },
+            body: `platform=www&brief_column_id=${id}&last_id=`,
+        })
+        .json();
+
+    const list = response.data.datalist;
+
+    const items = await Promise.all(
+        list.map(async (item) => {
+            const link = `https://www.huxiu.com/brief/${item.brief_id}`;
+
+            const detailResponse = await got.get(link);
+            const $ = cheerio.load(detailResponse.data);
+
+            const description = $('#js-part').html();
+
+            return {
+                title: item.title,
+                pubDate: item.format_publish_time.includes('前') ? parseRelativeDate(item.format_publish_time) : parseDate(item.format_publish_time),
+                description,
+                link,
+            };
+        })
+    );
+
+    const info = `虎嗅 - ${'虎嗅早报'}`;
```

Comment:
```suggestion
    const info = '虎嗅 - 虎嗅早报';
```

Reply:
I fix it in the last commit.
I forget to get the title and replace it.